bench_test.go

```diff
@@ -0,0 +1,179 @@
+package dhstore_test
+
+import (
+	"bytes"
+	"context"
+	crand "crypto/rand"
+	"encoding/json"
+	"io"
+	"log"
+	"math"
+	"math/rand"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/ipni/dhstore"
+	"github.com/ipni/dhstore/metrics"
+	"github.com/multiformats/go-multihash"
+	"github.com/stretchr/testify/require"
+)
+
+func BenchmarkDHStore_PutMultihashes_0(b *testing.B) {
+	benchmarkPut(b, 300, 1000, 200)
+}
+
+func BenchmarkDHStore_GetMultihashes_0(b *testing.B) {
+	benchmarkGet(b, 500, 1000, 200)
+}
+
+func benchmarkPut(b *testing.B, chunks, chunkSize, vkLen int) {
+	b.SetBytes(int64(34 + vkLen))
```

Comment:
Where did 34 come from? I recommend leaving an inline comment explaining the rationale if it cannot be calculated from given parameters programaticelly.